src/Writer/Deleted.php

```diff
@@ -32,14 +32,15 @@ class Deleted
     /**
      * Set the feed character encoding
      *
-     * @param  string $encoding
-     * @return null|string
-     * @return Deleted
+     * @param null|string $encoding
```

Comment:
Imho we should not allow `null` at all as it was not previously declared like that (or is that a conflict with the implementing interface?).

Even if there is code which passes `null` to this method in some cases, we should let the `!is_string` do its job.
Please revert this change and add errors regarding to the old annotation to the baseline.

Reply:
Have reverted this to how it was - it has 2 return annotations, which Psalm really doesn't like, heh. I'll fix in another PR.